cmd/http/main.go

```diff
@@ -121,6 +126,19 @@ func main() {
 		return hs.Start()
 	})
 
+	switch {
+	case httpServerConfig.CertFile != "" || httpServerConfig.KeyFile != "":
+		logger.Info(fmt.Sprintf("%s service https server listening at %s:%s with TLS cert %s and key %s", svcName, httpServerConfig.Host, httpServerConfig.Port, httpServerConfig.CertFile, httpServerConfig.KeyFile))
+		if err := proxyHTTPS(ctx, httpServerConfig, logger, svc); err != nil {
+			logger.Error(fmt.Sprintf("failed to start proxy server with error: %v", err))
+		}
+	default:
```

Comment:
Fix port mapping and URI
- This is the error from HTTP adapter
	```bash
	{"level":"error","message":"failed to start proxy server with error: parse \":81\": missing protocol scheme","ts":"2023-10-16T11:41:52.786991106Z"}
	```

- This is the error when trying to publish
	```bash
	unable to send messages to channels: HTTP failed to send message from thing 77f1cc97-2104-4ab0-be6e-5c88706b4196 to channel 0736b9f8-9293-49ec-bc92-4ff9550e33c3: Status: : Post "http://localhost:8008/channels/0736b9f8-9293-49ec-bc92-4ff9550e33c3/messages": read tcp [::1]:51840->[::1]:8008: read: connection reset by peer
	```